addons/web_editor/static/src/scss/bootstrap_overridden.scss

```diff
@@ -81,3 +81,7 @@ $gray-900: map-get($grays, '900') !default;
 $black: map-get($grays, 'black') !default;
 
 $o-color-system-initialized: true;
+
+.tooltip {
+    pointer-events: none !important;
+}
```

Comment:
The fix itself seems alright :wink: 
However, I think it should be done in another file, like `web_editor/static/src/scss/wysiwyg_snippets.scss` where we define the options styles :slightly_smiling_face: 
Also, the class selector should be more specific, to only target the tooltips that are in the right panel and not every tooltips in general, as only these ones are problematic :wink:

Reply:
It's an issue with the bootstrap tooltip itself and not a style preference thus I find it more relevant to fix bootstrap itself for the whole web editor (given that's a common issue for all tooltips)

Reply:
Hmm I guess it depends on if we want to fix all of them or just solve the issue of the options flicker :thinking: 
I looked a bit and didn't find anything broken by your fix so I think it can stay like this :)